package/wpe/wpeframework/wpeframework.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-WPEFRAMEWORK_VERSION = e3f46e7eede8457e11c9af5215f25914e42b95e2 
+WPEFRAMEWORK_VERSION = 8c01c446e09ba6652783c0f2f40cd2aba5363089
```

Comment:
Same here, no need to bump unless it is behind. Seems to point at a hash on a branch?

Reply:
I will change this into hash of master branch